src/ui/public/vis/request_handlers/courier.js

```diff
@@ -29,7 +17,8 @@ const CourierRequestHandlerProvider = function (Private, courier, timefilter) {
       // Using callParentStartHandlers: true we make sure, that the parent searchSource
       // onSearchRequestStart will be called properly even though we use an inherited
       // search source.
-      const requestSearchSource = new SearchSource().inherits(searchSource, { callParentStartHandlers: true });
+      const timeFilterSearchSource = new SearchSource().inherits(searchSource, { callParentStartHandlers: true });
```

Comment:
I could see this working as

```js
const timeFilterSearchSource = searchSource.makeChild() // default to callParentStartHandlers: true
const requestSearchSource = timeFilterSearchSource.makeChild()
```

Reply:
i would keep any unnecessary/unrelated changes out of this PR as its getting big already

Reply:
ignore my previous comment. implemented